### DISCUSSION TITLE
Keyboard

### DISCUSSION THREAD

**Discussion by SGThirkell** [lllyasviel/Fooocus#2260] [Q&A] [answered]
Hi, has anyone had a glitch where some keystrokes are not working when entering prompts? 
S and R stopped working as I was entering a prompt today, with everything else still ok. Obviously, I've checked the keyboard and is everything is fully functioning

**Comment by docppp** [comment, marked as answer]
S and R are shortcuts in inpaint, so maybe you had focus on it?

**Reply by SGThirkell** [reply]
You may be on to something there. I can't re-create it as yet but at least I have a direction now. Thank you.
